cli/clistat/stat.go

```diff
@@ -0,0 +1,218 @@
+package clistat
+
+import (
+	"runtime"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/elastic/go-sysinfo"
+	"github.com/spf13/afero"
+	"golang.org/x/xerrors"
+	"tailscale.com/types/ptr"
+
+	sysinfotypes "github.com/elastic/go-sysinfo/types"
+)
+
+// Prefix is an SI prefix for a unit.
+type Prefix string
```

Comment:
I think changing the terminology from prefix to unit might be more flexible with whatever types of stats we add to this command in the future. I suppose usage would then rather be `--unit KiB` vs `--prefix Ki`. Not a requirement by any means, just a thought.